docs/pages/example/geojson-layer-in-stack.html

```diff
@@ -9,6 +9,13 @@
 
 map.on('load', function () {
 
+    // Find the index of the bottom-most type: symbol layer in the map style
+    // Insert the layer beneath any symbol layer.
+    var layers = map.getStyle().layers.reverse();
+    var labelLayerIdx = layers.findIndex(function (layer) {
```

Comment:
[`findIndex`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#Browser_compatibility) isn't available in IE, and don't you want to insert the layer before the first symbol layer, rather than before the last non-symbol layer?

```js
var before = undefined;
for (var i = 0; i < layers.length; i++) {
    if (layers[i].type === symbol) {
        before = layers[i].id;
        break;
    }
}
```

Reply:
🤦‍♀️ good catch(es) @jfirebaugh!